main.py

```diff
@@ -0,0 +1,171 @@
+import csv
+import json
+import datetime
+import config as config
+from youtubesearchpython import *
+import pandas as pd
+from telebot.callback_data import CallbackData, CallbackDataFilter
+from telebot import types, TeleBot, ContinueHandling
+from telebot.custom_filters import AdvancedCustomFilter
+from CallBackFilter import CallbackFilter
+
+from db import DB
+
+db = DB()
+db.setup()
+
+
+
+def csv_to_list(file_csv):
+    return list(csv.DictReader(open(file_csv, 'r')))
+
+
+def song_titles(file_csv):
+    df = pd.read_csv(file_csv)
+    return '\n'.join('\U00002b50 '+str(e) for e in list(df["track"]))
+
+KPOP = csv_to_list('Data/kpop.csv')
+ROCK = csv_to_list('Data/rock.csv')
+bts = song_titles('Data/bts.csv')
+exo = song_titles('Data/exo.csv')
+elton = song_titles('Data/elton.csv')
+queen = song_titles('Data/queen.csv')
+
+kpops_factory = CallbackData('kpops_id', prefix='kpops')
+rocks_factory = CallbackData('rocks_id', prefix='rocks')
+bot = TeleBot(config.API_TOKEN)
+markup = types.ReplyKeyboardMarkup(row_width=2)
+btn1 = types.KeyboardButton('KPOP')
+btn2 = types.KeyboardButton('ROCK')
+
+markup.add(btn1, btn2)
+now = datetime.datetime.now()
+
+def greetings():
+    greeting = ""
+    if 5 <= now.hour < 12:
+        greeting = "Good morning"
+    elif 12 <= now.hour < 17:
+        greeting = "Good afternoon"
+    elif 17 <= now.hour <= 23:
+        greeting = "Good evening"
+    return greeting
```

Comment:
5, 12, 17, and 23 - are hardcoded numbers.
Replace them

Reply:
@HenintsoaHARINORO haven't fixed this one.
Move to constant with mimingful names.

Reply:
fixed